docs/basics.md

```diff
@@ -82,6 +82,7 @@ Frontends can be defined using the following rules:
 - `HostRegexp: traefik.io, {subdomain:[a-z]+}.traefik.io`: Adds a matcher for the URL hosts. It accepts templates with zero or more URL variables enclosed by `{}`. Variables can define an optional regexp pattern to be matched.
 - `Method: GET, POST, PUT`: Method adds a matcher for HTTP methods. It accepts a sequence of one or more methods to be matched.
 - `Path: /products/, /articles/{category}/{id:[0-9]+}`: Path adds a matcher for the URL paths. It accepts templates with zero or more URL variables enclosed by `{}`.
+- `PathReplace: /serverless-path`: Replaces the path and adds the old path to the `X-Replaced-Path` header. Useful for mapping to AWS Lambda or Google Cloud Functions.
```

Comment:
I'm not sure if it was a conscious decision, but we seem to have a pattern that `Path*` match a path and optionally modify it (e.g., `Path`, `PathPrefix`, `PathPrefixStrip`), while `*Path` do modifications only (`AddPrefix`). Regardless of how deliberate this naming choice was, I like it. :-)

Your new rule seems to fall into the second category. If that was your intention too, then I'd suggest to call it `ReplacePath` for naming consistency reasons.